freerad_manager/django_freeradius/models.py

```diff
@@ -1,5 +1,5 @@
 import csv
-from base64 import encodestring
+from base64 import encodebytes as encodestring
```

Comment:
Is this change retrocompatibile with py37 and py38?

Reply:
If not

Protect the import with an if that checks the python version

Reply:
ok which PYthon VErsion above you want me to use this

Reply:
no worries, my bad, encodestyring was deprecated from 3.1 and 3.7 already has encodestring, so it is good as it is
thank you